packages/console/src/pages/TenantSettings/index.module.scss

```diff
@@ -1,9 +1,8 @@
 @use '@/scss/underscore' as _;
 
 .container {
-  overflow-y: auto;
```

Comment:
This should still be needed in case you have many tenants that piling up higher than your available height in browser? You will not be able to scroll if you remove this line

Reply:
use flex-grow instead of overflow-y since this should lead to a global scrollbar.